app.js

```diff
@@ -25,7 +25,7 @@ app.use('/admin',adminData.routes);
 app.use(shopRoutes);
 
 app.use((req,res,next) => {
-    res.status(404).render('404')
+    res.status(404).render('404',{pageTitle : 'Error 404'});
```

Comment:
Don't you think its better if the page title is `Not Found` or something more non-technical and its content might contain some status codes and all?

Reply:
Okay . will update it . status code already sent back in response.

Reply:
Done , Now pageTitle is Page Not Found .